app/outliers.py

```diff
@@ -21,7 +21,6 @@
 from analyzers.terms import TermsAnalyzer
 from analyzers.word2vec import Word2VecAnalyzer
 
-
 EE_OUTLIERS_VERSIONS = "0.2.8"
```

Comment:
This version variable still needs to be updated to 0.2.9.
It will be easier to only have to update the version number in one place, so maybe read it from the file or let jenkins set it correctly?